packages/base-controller/CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [4.0.1]
+### Changed
+- Remove `subscribe` property from `BaseControllerV2` ([#3590](https://github.com/MetaMask/core/pull/3590))
+  - This is non-breaking because previously it was set to `undefined`. Accessing it still gets the same result.
```

Comment:
If you have a BaseControllerV2 controller and you attempt to access `subscribe`, you will now get a type error where previously you wouldn't.  I think we've been treating this sort of thing as breaking because it forces a change to code.

Reply:
True.... hmm.

We could restore it, and deprecate the property instead, to make this non-breaking. It would seem silly to introduce another base controller breaking change just for this change.

Reply:
Alright, that sounds like a good solution.

Reply:
Done here: https://github.com/MetaMask/core/pull/3698

Reply:
Alright, change has been incorporated, and changelog updated in https://github.com/MetaMask/core/pull/3695/commits/a46f115e65b0cf20351942e95f4bc46d8e6934d5